mindsdb/api/http/namespaces/file.py

```diff
@@ -177,23 +197,7 @@ def on_file(file):
                 if ctx.user_class != 1:
                     info = requests.head(url, timeout=30)
                     file_size = info.headers.get("Content-Length")
-                    try:
-                        file_size = int(file_size)
-                    except Exception:
-                        pass
-
-                    if file_size is None:
-                        return http_error(
```

Comment:
`_validate_file_size` won't raise error if file_size is None, should it be like this?